Document/0x04g-Testing-Cryptography.md

```diff
@@ -98,6 +99,11 @@ Inventing proprietary cryptographic functions is time consuming, difficult, and
 
 Carefully inspect all the cryptographic methods used within the source code, especially those that are directly applied to sensitive data. All cryptographic operations should use standard cryptographic APIs for Android and iOS (we'll write about those in more detail in the platform-specific chapters). Any cryptographic operations that don't invoke standard routines from known providers should be closely inspected. Pay close attention to standard algorithms that have been modified. Remember that encoding isn't the same as encryption! Always investigate further when you find bit manipulation operators like XOR (exclusive OR).
 
+At all implementations of cryptography, you need to ensure that the following always takes place:
+- Worker keys (like intermediary/derived keys in AES/DES/Rijndael) are propperly removed from memory after consumption.
+- The inner state of a cipher should be kept as short in memory as possible.
```

Comment:
should be kept as short in memory as possible -> 
Alt.1 : should be kept in memory as short as possible
Alt.2: should be removed from memory as soon as possible